### PR TITLE
Add CORS faq

### PR DESCRIPTION
**Pull request by howardjohn**
This is very commonly confused topic, so having a page will hopefully
help or provide a simple link to users struggling.

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
cc @craigbox

**Review by mandarjog — APPROVED** [review]
Thanks john

File: content/en/about/faq/traffic-management/cors.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+After applying [CORS configuration](/docs/reference/config/networking/virtual-service/#CorsPolicy), you may find that seemingly nothing happened and wonder what went wrong.
+CORS is a commonly misunderstood HTTP concept that often leads to confusion when configuring.
+
+To understand this, it helps to take a step back and look at [what CORS is](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) and when it should be used.
+By default, browsers have restrictions on "cross origin" requests initiated by scripts.
+This prevents, for example, a website `attack.example.com` from making a JavaScript request to `bank.example.com` and stealing a users sensitive information.
+
+In order to allow this request, `bank.example.com` must allowlist `attack.example.com` to be able to perform these cross origin requests.
```
Comment (frankbu) [inline content/en/about/faq/traffic-management/cors.md:13]:
```suggestion
In order to allow this request, `bank.example.com` must allow `attack.example.com` to perform cross origin requests.
```

File: content/en/about/faq/traffic-management/cors.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+This is where CORS comes in place. If we were serving `bank.example.com` in an Istio enabled cluster, we could configure a `corsPolicy` to allow this:
```
Comment (frankbu) [inline content/en/about/faq/traffic-management/cors.md:14]:
```suggestion
This is where CORS comes in. If we were serving `bank.example.com` in an Istio enabled cluster, we could configure a `corsPolicy` to allow this:
```

File: content/en/about/faq/traffic-management/cors.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      allowOrigins:
+      - exact: https://attack.example.com
+...
+{{< /text >}}
+
+In this case we explicitly allow a single origin; wildcards are common for non-sensitive pages.
+
+Once we do this, a common mistake is to send a request like `curl bank.example.com -H "Origin: https://attack.example.com"`, and expect the request to be rejected.
+However, curl and many other clients will not see a rejected request, because CORS is a browser constraint.
+The CORS configuration simple adds `Access-Control-*` headers in the response; it is up to the client (browser) to reject the request if the response is not satisfactory.
```
Comment (frankbu) [inline content/en/about/faq/traffic-management/cors.md:35]:
```suggestion
The CORS configuration simply adds `Access-Control-*` headers in the response; it is up to the client (browser) to reject the request if the response is not satisfactory.
```

**Review by frankbu — APPROVED** [review]

**Pull request merged**